src/services/FeedProcessor/UniswapV3MultiProcessor.ts

```diff
@@ -13,7 +13,7 @@ interface FeedFetcherParams {
 }
 
 @injectable()
-export default class UniswapV3MultiProcessor implements FeedFetcherInterface {
+export default class UniswapV3MultiProcessor {
```

Comment:
please do not remove `FeedFetcherInterface`

Reply:
I will revert it.
The reason why I removed it, was that this class is a multi-processor and not a fetcher as some others.

In case you also want, I could create another interface for the other multiprocessors that satificies that the apply method takes an array of FeedFetchers.

Reply:
we definately need to use some interface that will be apply  everywhere, otherwize it will a mess soon

Reply:
Yes, I just mean another interface: `FeedProcessorInterface` or something in that style. 

My point is that a feed multiprocessor is not the same as a feed fetcher.